config/initializers/content_security_policy.rb

```diff
@@ -42,6 +42,7 @@
     policy.script_src :self,
       :https,
       :unsafe_inline,
+      :unsafe_eval,
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.